src/entrypoints/background.ts

```diff
@@ -1,6 +1,11 @@
-import type { BetterBuff } from "@/lib/@types/BetterBuff";
+import type { BetterBuff } from '@/lib/@types/BetterBuff';
+import buffItemsSorted from '@/assets/buff-items-sorted.json';
+import { browser } from 'wxt/browser';
+import { defineBackground } from 'wxt/sandbox';
 
 export default defineBackground(() => {
+    const buffItems: BetterBuff.BuffItemEntry = buffItemsSorted;
```

Comment:
I couldn't find a way to directly type the imported `buffItemsSorted` variable, so this is a workaround.

Reply:
If we add `"resolveJsonModule": true` in the `tsconfig.json`, imported JSON should be typed automatically. In TS this works similar to defining something `as const`.
I just adapted the PR accordingly, take a look at commit bf50be8.

Reply:
Nice, thank you :)